internal/converter/types.go

```diff
@@ -72,7 +72,7 @@ func (c *Converter) registerType(pkgName string, msgDesc *descriptor.DescriptorP
 }
 
 // Convert a proto "field" (essentially a type-switch with some recursion):
-func (c *Converter) convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, msgDesc *descriptor.DescriptorProto, duplicatedMessages map[*descriptor.DescriptorProto]string, messageFlags ConverterFlags) (*jsonschema.Type, error) {
+func (c *Converter) convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, msgDesc *descriptor.DescriptorProto, duplicatedMessages map[*descriptor.DescriptorProto]string, enums map[*descriptor.EnumDescriptorProto]string, messageFlags ConverterFlags) (*jsonschema.Type, error) {
```

Comment:
`duplicatedMessages` (i have no idea why it is named that) contains all the protos defined and used within a top level message. 
The enums map is the equivalent for enums (see below for where I created it)